### ISSUE TITLE
Epic: Tasks UI Enhancements

### ISSUE THREAD

**Issue by ALuckette** [influxdata/ui#6125]

**Comment by ALuckette** [comment]
Related to https://github.com/influxdata/Analytics/issues/283

**Comment by ALuckette** [comment]
Sharing @mavarius @juliajames12 @kristinarobinson 

This is the Epic for the MVP Tasks UI. We'll update the ACs in this ticket prior to moving the work into development.

**Comment by hoorayimhelping** [comment]
## Thoughts

- When implementing the new tasks page, consider how we might prefill some of the form fields based on data based in via the URL. 

- When Figma designs show Clockface 4 components - don't worry about using Clockface 4 if it's not ready by implementation time. Just use the Clockface component and when Clockface 4 drops, these will automatically get updated

**Comment by aanthony1243** [comment]
I've stated in slack, but to clarify here.  Please add Gavin Cabbage to the engineering portion of the team. 

I like the progress here, but there are some issues we need to work out: 
1.  How are we resolving with users' tasks that don't have a script behind them?   I would say that there are at least tens of thousands of these legacy tasks. When they view the task code, will they have a different experience?  Can they still create a legacy task in the UI?  If they can still create one by the API, users may use the CLI tooling to create tasks from their development workflow, and then view them in the UI.  
2. When the user is taken to the editor, are they properly instructed about the task.every and task.offset parameters?  will we inject that code for them?  
3. What if a user is in the script editor and they want to schedule an existing script as a new task?  It feels odd to have to "save" a script in order to access the task creation function.  

We should also note that the new editor is a beta feature, and so this feature would have to be blocked by whether or not the new editor is activated.   Ideally, we would get the new editor out of beta before we deliver this work.
